src/components/Home/Home.jsx

```diff
@@ -10,22 +10,23 @@ import PapersList from 'src/components/Papers/PapersList'
 import PlaceholdersList from 'src/components/Placeholders/PlaceholdersList'
 import HomeCloud from 'src/assets/icons/HomeCloud.svg'
 
-const Home = ({ data }) => {
+const Home = ({ allPapers }) => {
```

Comment:
I don't know if allPapers if the right name, why not `papers`?

Reply:
C'est surtout pour marquer un peu plus la différence avec le singulier, non présent sur cette page mais comme il a ce nom à différents endroits plus pertinents.. :)